builders/xcm/fees.md

```diff
@@ -0,0 +1,230 @@
+- **`BuyExecution(fees: MultiAsset, weight_limit: Option<Weight>)`** - takes asset(s) from holding to pay for execution fees. The amount that is actually used is determined by the interpreting system. If the fees are more than the specified amount then the instruction will result in error. You can optionally provide the amount of weight to be purchased. If the given amount of weight is lower than the estimated weight, an error will be thrown
+- **`DepositAsset(assets: MultiAssetFilter, max_assets: Compact, beneficiary: MultiLocation`** - removes asset(s) from holding to be sent to a destination account. The maximum assets defines the maximum number of unique assets to remove from holding, any remaining assets stay in holding
+
+## Calculation of Fees for Native Assets {: #calculation-of-fees-native }
+
+Substrate has introduced a weight system that determines how heavy an extrinsic is. When it comes to paying fees, users will pay a transaction fee that is proportionate to the weight of the call that is being made. One unit of weight is defined as one picosecond of execution time.
+
+The following sections will break down how to calculate XCM fees for Polkadot, Kusama, and Moonbeam-based networks. It's important to note that Kusama uses benchmarking and database read and write weight costs to determine the total weight costs for XCM instructions. Whereas Polkadot and Moonbeam-based networks use a fixed weight cost in their calculations. 
+
+There are two databases available in Polkadot and Kusama, RocksDB (which is the default) and ParityDB, both of which have their own associated weight costs for each network. 
+
+### Polkadot {: #polkadot }
+
+As previously mentioned, Polkadot currently uses a fixed amount of weight for all XCM instructions, which is `1,000,000,000` weight units.
+
+Although Polkadot doesn't currently use database weight units to calculate costs, the weight units for database operations are shared here for reference.
+
+|     Database      |    Read    |   Write    |
+|:-----------------:|:----------:|:----------:|
+| RocksDB (default) | 20,499,000 | 83,471,000 |
+|     ParityDB      | 11,826,000 | 30,052,000 |
+
+With the instruction weight cost established, you can calculate the cost of the instruction in DOT. 
+
+In Polkadot, the `ExtrinsicBaseWeight` is set to `85,212,000` which is mapped to 1/10 cents. Where 1 cent is `10^10 / 10,000`. Therefore, the formula for calculating the final fee in DOT looks like this:
+
+```
+Total-Planck-DOT = TotalWeight * PlanckDOT * (1 / ExtrinsicBaseWeight)
+DOT = Total-Planck-DOT / 10^10
+```
+
+Therefore, the actual calculation for the `WithdrawAsset` instruction is:
+
+```
+Total-Planck-DOT = 1000000000 * (10^10 / 1000) * (1 / 85212000)
+DOT = 117354360 / 10^10
+```
+
+The total cost is `0.0117354363 DOT`.
```

Comment:
Add an example of the weights and DOT cost of a XCM Token transfer to Polkadot (same as we did in the learning session). You can probably do this in a table?